frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -210,74 +230,97 @@ const EnvironmentAccordionBody = ({
                     )}
                 />
                 <ConditionallyRender
-                    condition={strategies.length > 0}
+                    condition={releasePlans.length > 0 || strategies.length > 0}
                     show={
-                        <ConditionallyRender
-                            condition={
-                                strategies.length < 50 ||
-                                !manyStrategiesPagination
-                            }
-                            show={
-                                <>
-                                    {strategies.map((strategy, index) => (
-                                        <StrategyDraggableItem
-                                            key={strategy.id}
-                                            strategy={strategy}
-                                            index={index}
-                                            environmentName={
-                                                featureEnvironment.name
-                                            }
-                                            otherEnvironments={
-                                                otherEnvironments
+                        <>
+                            {releasePlans.map((plan) => (
+                                <ReleasePlan key={plan.id} plan={plan} />
+                            ))}
+                            <ConditionallyRender
+                                condition={
+                                    releasePlans.length > 0 &&
+                                    strategies.length > 0
+                                }
+                                show={
+                                    <SectionSeparator>
+                                        <StyledBadge>OR</StyledBadge>
+                                    </SectionSeparator>
+                                }
+                            />
+                            <ConditionallyRender
```

Comment:
Same question around these being moved away from and consistency as further down